File: HelpMEApp/HelpMEServices/src/HelpMEServices/Models/Message.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace HelpMEServices.Models
{
    public class Message
    {
        public int Id { get; set; }
        public bool Seen { get; set; }
        public string UserSenderName { get; set; }
        public string UserSenderPhoto { get; set; }
        public string MessageText { get; set; }
        public DateTime TimeCreated { get; set; }

        [ForeignKey("ConversationId")]
        public int ConversationId { get; set; }
        public Conversation Conversation { get; set; }
    }
}

File: HelpMEApp/HelpMEWebPart/src/app/model/Message.ts
/**
 * Created by Hrvoje on 24.05.2016..
 */

export class Message{

  public Id: number;
  public Seen: boolean;
  public UserSenderName: string;
  public UserSenderPhoto: string;
  public UserRecieverName: string;
  public UserRecieverPhoto: string;
  public MessageText: string;
  public TimeCreated: any;


  constructor(Id:number, Seen:boolean, UserSenderName:string, UserSenderPhoto:string, UserRecieverName:string, UserRecieverPhoto:string, MessageText:string, TimeCreated:any) {
      this.Id = Id;
      this.Seen = Seen;
      this.UserSenderName = UserSenderName;
      this.UserSenderPhoto = UserSenderPhoto;
      this.UserRecieverName = UserRecieverName;
      this.UserRecieverPhoto = UserRecieverPhoto;
      this.MessageText = MessageText;
      this.TimeCreated = TimeCreated;
      }
}
